tecken/settings.py

```diff
@@ -421,6 +421,12 @@ def filter(self, record):
         },
     }
 
+CLOUD_SERVICE_PROVIDER = _config(
```

Comment:
I had a few questions here:
* Is there an optimal location within this `settings.py` file to add this new env var?
* Is there a way to enforce that this config is an enum (i.e. one of "AWS" or "GCP")? I realize we may do this in the code downstream.
* I noticed in the Tecken docs that this env var is listed as "Required: No". I don't know how to configure that or if I need to. I only see a small number of env vars listed as "Required: Yes".